src/pages/Contactpage.jsx

```diff
@@ -1,5 +1,54 @@
+import { useState } from "react";
 import PageNav from "../components/PageNav";
+
 function Contactpage() {
+  const [formData, setFormData] = useState({
+    email: "",
+    subject: "",
+    message: "",
+  });
+
+  const [statusMessage, setStatusMessage] = useState("");
+
+  // Handle form input change
+  const handleChange = (e) => {
+    setFormData({
+      ...formData,
+      [e.target.id]: e.target.value,
+    });
+  };
+
+  // Function to submit form data to the API
+  const handleSubmit = async (e) => {
+    e.preventDefault();
+
+    // Create payload to send to the backend
+    const payload = {
+      subject: formData.subject,
+      email: formData.email,
+      message: formData.message,
+    };
+
+    try {
+      const response = await fetch("http://localhost:8080/notification/email", {
```

Comment:
Don't do this.
Put base URL in config .env file and API endpoint in one constant file.